cfme/tests/test_manual.py

```diff
@@ -5164,120 +4848,35 @@ def test_osp_test_creating_multiple_migration_plans_with_same_name():
 
 
 @pytest.mark.manual
-@test_requirements.service
-@pytest.mark.tier(3)
-def test_database_wildcard_should_work_and_be_included_in_the_query():
+@test_requirements.ansible
```

Comment:
missing `tier` here? or will this be addressed by the FAs?

Reply:
This is not my FA, I have not added anything there and actually I have deleted them, I have no idea that this added TCs mean and their tier severity.